frontend/App.tsx

```diff
@@ -29,6 +30,9 @@ export default DrawerNavigator({
   Conversation:  {
     screen: Conversation,
   },
+  ConversationHistory:  {
```

Comment:
Warum hast du die History in einen neuen Screen gepackt? Wäre es nicht besser nur ein Conversation-Screen zu haben?

Reply:
Nachdem ich den History Screen in seiner Form eher als einen "Debug"-Screen von Seiten der POs verstanden habe, dachte ich mir dass hier eine räumliche Trennung vielleicht schöner wäre. Notfalls ist das ja kein Aufwand diesen mit in Conversation einzugliedern.

Reply:
Ich dachte es sei nicht nur ein Debug-Screen! Vielleicht habe ich das verpasst - klingt daher gut 👍